### ISSUE TITLE
All `@create-figma-plugin/*` modules are currently broken

### ISSUE THREAD

**Issue by yuanqing** [yuanqing/create-figma-plugin#8]
Not sure if it’s a problem with the `npm` registry

**Comment by yuanqing** [comment]
`build`, `common` and `utilities` all show an incorrect latest version in the `npm` CLI:

- `@create-figma-plugin/build` — `0.0.59`
- `@create-figma-plugin/common` — `0.0.59`
- `@create-figma-plugin/ui` — `0.0.60`
- `@create-figma-plugin/utilities` — `0.0.59`
- `create-figma-plugin` — `0.0.60`

---

# `@create-figma-plugin/build`

```
$ npm view @create-figma-plugin/build

@create-figma-plugin/build@0.0.59 | MIT | deps: 17 | versions: 58
Build a Figma plugin
https://github.com/yuanqing/create-figma-plugin#readme

keywords: create-figma-plugin, figma, figma-plugin, figma-plugins

bin: build-figma-plugin

dist
.tarball: https://registry.npmjs.org/@create-figma-plugin/build/-/build-0.0.59.tgz
.shasum: a38147cc0e8aa5585e2494798656ad84bed5886b
.integrity: sha512-uIg3JCkcvwv116RkU1UuFIRELWFD+XaJ9wKdQDtosmaxdamoCzMPOiB8OM9HpsuOpCoSE6jiGe5S5MNW/VhLMQ==
.unpackedSize: 12.4 kB

dependencies:
@babel/core: ^7.9.0
@babel/plugin-proposal-object-rest-spread: ^7.9.0
@babel/plugin-transform-react-jsx: ^7.9.4
@babel/plugin-transform-template-literals: ^7.8.3
@create-figma-plugin/common: ^0.0.59
@create-figma-plugin/utilities: ^0.0.59
babel-loader: ^8.1.0
chokidar: ^3.3.1
css-loader: ^3.4.2
find-up: ^4.1.0
fs-extra: ^9.0.0
sade: ^1.7.3
sass-loader: ^8.0.2
style-loader: ^1.1.3
temp-write: ^4.0.0
terser-webpack-plugin: ^2.3.5
webpack: ^4.42.1

maintainers:
- yuanqing <desk@yuanqing.sg>

dist-tags:
latest: 0.0.59  

published 42 minutes ago by yuanqing <desk@yuanqing.sg>
```

# `@create-figma-plugin/common`

```
$ npm view @create-figma-plugin/common

@create-figma-plugin/common@0.0.59 | MIT | deps: 4 | versions: 58
Internal code shared across the `create-figma-plugin` monorepo
https://github.com/yuanqing/create-figma-plugin#readme

keywords: create-figma-plugin, figma, figma-plugin, figma-plugins

dist
.tarball: https://registry.npmjs.org/@create-figma-plugin/common/-/common-0.0.59.tgz
.shasum: e3f092f4f33f28e782d17bebf225f2267065c385
.integrity: sha512-3qxdvTj4KDLT1KA8zNsLk9boys+S3LJ9PwpgCqmw+Ywy/7vjHioGLEMncZq6ir6iGFE8DJqnp92PLchZhHZlhg==
.unpackedSize: 6.8 kB

dependencies:
@sindresorhus/slugify: ^0.11.0 fs-extra: ^9.0.0               
chalk: ^4.0.0                  npmlog: ^4.1.2                 

maintainers:
- yuanqing <desk@yuanqing.sg>

dist-tags:
latest: 0.0.59  

published 42 minutes ago by yuanqing <desk@yuanqing.sg>
```

# `@create-figma-plugin/ui`

```
$ npm view @create-figma-plugin/ui

@create-figma-plugin/ui@0.0.60 | MIT | deps: 2 | versions: 29
A Preact implementation of the Figma design system for use in Figma plugins
https://github.com/yuanqing/create-figma-plugin#readme

keywords: component, component-library, create-figma-plugin, design-system, figma, figma-plugin, figma-plugin-ds, figma-plugins, figma-ui, form, preact, ui

dist
.tarball: https://registry.npmjs.org/@create-figma-plugin/ui/-/ui-0.0.60.tgz
.shasum: f9209accfec02bb42ae68709e44c9ad7d97542d6
.integrity: sha512-OXpAfb1LMggX+5ipclcQ88HQY/3ALMhegUq02bTZOMX/2QEbvzRVNwfefU8lUqsUXTujtzx1oommOcmWQVZ3EQ==
.unpackedSize: 124.6 kB

dependencies:
@create-figma-plugin/utilities: ^0.0.60 @sindresorhus/class-names: ^1.1.0       

maintainers:
- yuanqing <desk@yuanqing.sg>

dist-tags:
latest: 0.0.60  

published 10 minutes ago by yuanqing <desk@yuanqing.sg>
```

# `@create-figma-plugin/utilities`

```
$ npm view @create-figma-plugin/utilities

@create-figma-plugin/utilities@0.0.59 | MIT | deps: none | versions: 46
Helper functions for developing Figma plugins
https://github.com/yuanqing/create-figma-plugin#readme

keywords: create-figma-plugin, figma, figma-plugin, figma-plugins

dist
.tarball: https://registry.npmjs.org/@create-figma-plugin/utilities/-/utilities-0.0.59.tgz
.shasum: 210a16d91369a808309620673419eea95c49f649
.integrity: sha512-DXBRduQiIWTmj1N2CeT/0+8IVwlhovV4nDkkw5v5zui/i/8073uNyG/v249QLU5E6utxSEj8F7NMTe1sZF3iDQ==
.unpackedSize: 16.8 kB

maintainers:
- yuanqing <desk@yuanqing.sg>

dist-tags:
latest: 0.0.59  

published 42 minutes ago by yuanqing <desk@yuanqing.sg>
```

# `create-figma-plugin`

```
$ npm view create-figma-plugin

create-figma-plugin@0.0.60 | MIT | deps: 10 | versions: 60
Scaffold a new Figma plugin
https://github.com/yuanqing/create-figma-plugin#readme

keywords: create-figma-plugin, figma, figma-plugin, figma-plugins

bin: create-figma-plugin

dist
.tarball: https://registry.npmjs.org/create-figma-plugin/-/create-figma-plugin-0.0.60.tgz
.shasum: 766551f2d4f66edf1a467355f9c35f88efadeab1
.integrity: sha512-7j4n5PwYGB6T6NR13NwXKI2RJfg62i+zHxd1aBboC524ry9k9NndNBh10DRVQCgtQ+qGAb2aZNtyFMqxb8sM9Q==
.unpackedSize: 13.4 kB

dependencies:
@create-figma-plugin/common: ^0.0.60 globby: ^11.0.0                      
capital-case: ^1.0.3                 inquirer: ^7.1.0                     
degit: ^2.8.0                        is-utf8: ^0.2.1                      
fs-extra: ^9.0.0                     mustache: ^4.0.1                     
git-user-name: ^2.0.0                sade: ^1.7.3                         

maintainers:
- yuanqing <desk@yuanqing.sg>

dist-tags:
latest: 0.0.60  

published 10 minutes ago by yuanqing <desk@yuanqing.sg>
```

**Comment by yuanqing** [comment]
Output after I ran `yarn run publish` locally:

```
$ yarn run publish
yarn run v1.22.4
$ yarn run build && cpy 'docs/*.md' README.md packages/create-figma-plugin --parents && lerna publish from-package
$ yarn run clean && lerna exec --scope '@create-figma-plugin/{build,common}' --scope create-figma-plugin 'babel src --copy-files --delete-dir-on-start --out-dir lib --root-mode upward'
$ rimraf .nyc_output '*.log' 'packages/*/lib' 'packages/**/fixtures/**/{build,figma-plugin,manifest.json,node_modules}' 'packages/create-figma-plugin/{docs,README.md}' packages/ui/build
lerna notice cli v3.20.2
lerna notice filter including ["@create-figma-plugin/{build,common}","create-figma-plugin"]
lerna info filter [ '@create-figma-plugin/{build,common}', 'create-figma-plugin' ]
lerna info Executing command in 3 packages: "babel src --copy-files --delete-dir-on-start --out-dir lib --root-mode upward"
Successfully compiled 5 files with Babel.
Successfully compiled 6 files with Babel.
Successfully compiled 6 files with Babel.
lerna success exec Executed command in 3 packages: "babel src --copy-files --delete-dir-on-start --out-dir lib --root-mode upward"
lerna notice cli v3.20.2
lerna WARN Yarn's registry proxy is broken, replacing with public npm registry
lerna WARN If you don't have an npm token, you should exit and run `npm login`
lerna info publish rooted leaf detected, skipping synthetic root lifecycles

Found 5 packages to publish:
 - @create-figma-plugin/build => 0.0.60
 - @create-figma-plugin/common => 0.0.60
 - create-figma-plugin => 0.0.60
 - @create-figma-plugin/ui => 0.0.60
 - @create-figma-plugin/utilities => 0.0.60

? Are you sure you want to publish these packages? Yes
lerna info publish Publishing packages to npm...
lerna info Verifying npm credentials
lerna http fetch GET 200 https://registry.npmjs.org/-/npm/v1/user 909ms
lerna http fetch GET 200 https://registry.npmjs.org/-/org/yuanqing/package?format=cli 968ms
lerna info Checking two-factor auth mode
lerna http fetch GET 200 https://registry.npmjs.org/-/npm/v1/user 914ms
lerna success published @create-figma-plugin/utilities 0.0.60
lerna notice 
lerna notice 📦  @create-figma-plugin/utilities@0.0.60
lerna notice === Tarball Contents === 
lerna notice 602B  src/layers/collapse-layer.js                              
lerna notice 625B  src/layers/compute-bounding-box.js                        
lerna notice 653B  src/layers/compute-maximum-bounds.js                      
lerna notice 471B  src/layers/create-image-paint.js                          
lerna notice 1.1kB src/events.js                                             
lerna notice 138B  src/layers/absolute-position/get-absolute-position.js     
lerna notice 538B  src/layers/get-layers/get-document-components.js          
lerna notice 209B  src/layers/get-layers/get-selected-layers-or-all-layers.js
lerna notice 768B  src/layers/group-sibling-layers.js                        
lerna notice 1.1kB src/index.js                                              
lerna notice 191B  src/layers/insert-layer/insert-after-layer.js             
lerna notice 196B  src/layers/insert-layer/insert-before-layer.js            
lerna notice 280B  src/layers/is-layer-within-instance.js                    
lerna notice 706B  src/layers/load-fonts-async.js                            
lerna notice 1.1kB src/number.js                                             
lerna notice 1.6kB src/object.js                                             
lerna notice 167B  src/layers/remove-duplicate-layers.js                     
lerna notice 594B  src/layers/absolute-position/set-absolute-position.js     
lerna notice 117B  src/layers/set-relaunch-button.js                         
lerna notice 370B  src/settings.js                                           
lerna notice 607B  src/layers/sort-layers/sort-layers-by-name.js             
lerna notice 667B  src/string.js                                             
lerna notice 392B  src/layers/traverse-layer.js                              
lerna notice 525B  src/ui.js                                                 
lerna notice 1.3kB src/layers/sort-layers/update-layers-sort-order.js        
lerna notice 732B  package.json                                              
lerna notice 1.1kB LICENSE.md                                                
lerna notice === Tarball Details === 
lerna notice name:          @create-figma-plugin/utilities          
lerna notice version:       0.0.60                                  
lerna notice filename:      create-figma-plugin-utilities-0.0.60.tgz
lerna notice package size:  6.1 kB                                  
lerna notice unpacked size: 16.8 kB                                 
lerna notice shasum:        a5f2544c9ad3c273979b08eccfff4c5d72f2da5a
lerna notice integrity:     sha512-S10GUwMKi9lqf[...]ocELZf5Mwalgg==
lerna notice total files:   27                                      
lerna notice 
lerna http fetch PUT 200 https://registry.npmjs.org/@create-figma-plugin%2futilities 5587ms
lerna success published @create-figma-plugin/common 0.0.60
lerna notice 
lerna notice 📦  @create-figma-plugin/common@0.0.60
lerna notice === Tarball Contents === 
lerna notice 564B  lib/constants.js        
lerna notice 921B  lib/index.js            
lerna notice 750B  lib/log.js              
lerna notice 2.7kB lib/read-config-async.js
lerna notice 71B   lib/silence-logging.js  
lerna notice 762B  package.json            
lerna notice 1.1kB LICENSE.md              
lerna notice === Tarball Details === 
lerna notice name:          @create-figma-plugin/common             
lerna notice version:       0.0.60                                  
lerna notice filename:      create-figma-plugin-common-0.0.60.tgz   
lerna notice package size:  2.6 kB                                  
lerna notice unpacked size: 6.8 kB                                  
lerna notice shasum:        8715eb68a1a8bd3f36867b83c69bf4f681ae81c1
lerna notice integrity:     sha512-dr5/7/sBjD07L[...]HwZQcezdy5x8w==
lerna notice total files:   7                                       
lerna notice 
lerna http fetch PUT 200 https://registry.npmjs.org/@create-figma-plugin%2fcommon 6405ms
lerna success published @create-figma-plugin/ui 0.0.60
lerna notice 
lerna notice 📦  @create-figma-plugin/ui@0.0.60
lerna notice === Tarball Contents === 
lerna notice 1.7kB  src/components/button/button.js                                            
lerna notice 3.1kB  src/components/button/button.stories.js                                    
lerna notice 497B   src/components/icon/icons/check-circle-icon.js                             
lerna notice 175B   src/components/icon/icons/check-circle-icon.stories.js                     
lerna notice 235B   src/components/icon/icons/check-icon.js                                    
lerna notice 150B   src/components/icon/icons/check-icon.stories.js                            
lerna notice 1.7kB  src/components/checkbox/checkbox.js                                        
lerna notice 1.1kB  src/components/checkbox/checkbox.stories.js                                
lerna notice 537B   src/components/columns/columns.js                                          
lerna notice 2.0kB  src/components/columns/columns.stories.js                                  
lerna notice 404B   src/components/icon/icons/component-icon.js                                
lerna notice 166B   src/components/icon/icons/component-icon.stories.js                        
lerna notice 239B   src/components/textbox/utilities/compute-next-value.js                     
lerna notice 220B   src/components/container/container.js                                      
lerna notice 954B   src/components/container/container.stories.js                              
lerna notice 320B   src/components/icon/icons/cross-icon.js                                    
lerna notice 150B   src/components/icon/icons/cross-icon.stories.js                            
lerna notice 194B   src/components/divider/divider.js                                          
lerna notice 175B   src/components/divider/divider.stories.js                                  
lerna notice 5.4kB  src/components/dropdown-menu/dropdown-menu.js                              
lerna notice 2.3kB  src/components/dropdown-menu/dropdown-menu.stories.js                      
lerna notice 2.4kB  src/components/file-upload-button/file-upload-button.js                    
lerna notice 1.0kB  src/components/file-upload-button/file-upload-button.stories.js            
lerna notice 3.1kB  src/components/file-upload-dropzone/file-upload-dropzone.js                
lerna notice 256B   src/components/file-upload-dropzone/file-upload-dropzone.stories.js        
lerna notice 235B   src/components/icon/icons/frame-icon.js                                    
lerna notice 150B   src/components/icon/icons/frame-icon.stories.js                            
lerna notice 239B   src/components/icon/icon.js                                                
lerna notice 505B   src/components/icon/icons/image-icon.js                                    
lerna notice 150B   src/components/icon/icons/image-icon.stories.js                            
lerna notice 1.9kB  src/index.js                                                               
lerna notice 443B   src/components/inline/inline.js                                            
lerna notice 2.0kB  src/components/inline/inline.stories.js                                    
lerna notice 360B   src/components/textbox/utilities/is-keycode-character-generating.js        
lerna notice 294B   src/utilities/key-codes.js                                                 
lerna notice 842B   src/components/layer/layer.js                                              
lerna notice 2.7kB  src/components/layer/layer.stories.js                                      
lerna notice 573B   src/components/loading-indicator/loading-indicator.js                      
lerna notice 213B   src/components/loading-indicator/loading-indicator.stories.js              
lerna notice 308B   src/components/icon/icons/move-down-icon.js                                
lerna notice 163B   src/components/icon/icons/move-down-icon.stories.js                        
lerna notice 301B   src/components/icon/icons/move-right-icon.js                               
lerna notice 167B   src/components/icon/icons/move-right-icon.stories.js                       
lerna notice 244B   src/components/preview/preview.js                                          
lerna notice 484B   src/components/preview/preview.stories.js                                  
lerna notice 2.1kB  src/components/radio-buttons/radio-buttons.js                              
lerna notice 2.2kB  src/components/radio-buttons/radio-buttons.stories.js                      
lerna notice 210B   src/render.js                                                              
lerna notice 477B   src/components/icon/icons/search-icon.js                                   
lerna notice 154B   src/components/icon/icons/search-icon.stories.js                           
lerna notice 2.1kB  src/components/search-textbox/search-textbox.js                            
lerna notice 935B   src/components/search-textbox/search-textbox.stories.js                    
lerna notice 3.0kB  src/components/segmented-control/segmented-control.js                      
lerna notice 1.7kB  src/components/segmented-control/segmented-control.stories.js              
lerna notice 2.1kB  src/components/selectable-item/selectable-item.js                          
lerna notice 1.8kB  src/components/selectable-item/selectable-item.stories.js                  
lerna notice 249B   src/components/icon/icons/space-horizontal-icon.js                         
lerna notice 191B   src/components/icon/icons/space-horizontal-icon.stories.js                 
lerna notice 247B   src/components/icon/icons/space-vertical-icon.js                           
lerna notice 183B   src/components/icon/icons/space-vertical-icon.stories.js                   
lerna notice 490B   src/components/stack/stack.js                                              
lerna notice 1.7kB  src/components/stack/stack.stories.js                                      
lerna notice 2.8kB  src/components/tabs/tabs.js                                                
lerna notice 925B   src/components/tabs/tabs.stories.js                                        
lerna notice 547B   src/components/text/text.js                                                
lerna notice 388B   src/components/text/text.stories.js                                        
lerna notice 13.0kB src/components/textbox/textbox-autocomplete/textbox-autocomplete.js        
lerna notice 3.6kB  src/components/textbox/textbox-autocomplete/textbox-autocomplete.stories.js
lerna notice 5.3kB  src/components/textbox/textbox-numeric/textbox-numeric.js                  
lerna notice 2.9kB  src/components/textbox/textbox-numeric/textbox-numeric.stories.js          
lerna notice 1.9kB  src/components/textbox/textbox.js                                          
lerna notice 1.9kB  src/components/textbox/textbox.stories.js                                  
lerna notice 3.5kB  src/hooks/use-form.js                                                      
lerna notice 4.0kB  src/hooks/use-scrollable-menu.js                                           
lerna notice 229B   src/components/vertical-space/vertical-space.js                            
lerna notice 872B   src/components/vertical-space/vertical-space.stories.js                    
lerna notice 1.4kB  package.json                                                               
lerna notice 1.1kB  LICENSE.md                                                                 
lerna notice 37B    src/scss/base.scss                                                         
lerna notice 1.6kB  src/components/button/button.scss                                          
lerna notice 1.6kB  src/components/checkbox/checkbox.scss                                      
lerna notice 594B   src/components/columns/columns.scss                                        
lerna notice 1.0kB  src/scss/constants.scss                                                    
lerna notice 251B   src/components/container/container.scss                                    
lerna notice 113B   src/components/divider/divider.scss                                        
lerna notice 1.1kB  src/components/dropdown-menu/dropdown-menu.scss                            
lerna notice 1.1kB  src/components/file-upload-button/file-upload-button.scss                  
lerna notice 546B   src/components/file-upload-dropzone/file-upload-dropzone.scss              
lerna notice 83B    src/scss/font.scss                                                         
lerna notice 63B    src/components/icon/icon.scss                                              
lerna notice 448B   src/components/inline/inline.scss                                          
lerna notice 879B   src/components/layer/layer.scss                                            
lerna notice 369B   src/components/loading-indicator/loading-indicator.scss                    
lerna notice 264B   src/components/preview/preview.scss                                        
lerna notice 1.1kB  src/components/radio-buttons/radio-buttons.scss                            
lerna notice 442B   src/scss/reset.scss                                                        
lerna notice 793B   src/components/search-textbox/search-textbox.scss                          
lerna notice 1.5kB  src/components/segmented-control/segmented-control.scss                    
lerna notice 942B   src/components/selectable-item/selectable-item.scss                        
lerna notice 358B   src/components/stack/stack.scss                                            
lerna notice 751B   src/components/tabs/tabs.scss                                              
lerna notice 775B   src/components/text/text.scss                                              
lerna notice 627B   src/scss/text.scss                                                         
lerna notice 892B   src/components/textbox/textbox-autocomplete/textbox-autocomplete.scss      
lerna notice 1.2kB  src/components/textbox/textbox.scss                                        
lerna notice 236B   src/components/vertical-space/vertical-space.scss                          
lerna notice === Tarball Details === 
lerna notice name:          @create-figma-plugin/ui                 
lerna notice version:       0.0.60                                  
lerna notice filename:      create-figma-plugin-ui-0.0.60.tgz       
lerna notice package size:  23.6 kB                                 
lerna notice unpacked size: 124.6 kB                                
lerna notice shasum:        f9209accfec02bb42ae68709e44c9ad7d97542d6
lerna notice integrity:     sha512-OXpAfb1LMggX+[...]ommOcmWQVZ3EQ==
lerna notice total files:   106                                     
lerna notice 
lerna http fetch PUT 200 https://registry.npmjs.org/@create-figma-plugin%2fui 5748ms
lerna success published create-figma-plugin 0.0.60
lerna notice 
lerna notice 📦  create-figma-plugin@0.0.60
lerna notice === Tarball Contents === 
lerna notice 30B   plugin-templates/default/.gitignore       
lerna notice 590B  lib/cli.js                                
lerna notice 952B  lib/clone-from-template-async.js          
lerna notice 85B   plugin-templates/default/command.js       
lerna notice 2.0kB lib/create-figma-plugin-async.js          
lerna notice 653B  lib/install-dependencies-async.js         
lerna notice 1.3kB lib/interpolate-values-into-files-async.js
lerna notice 2.0kB lib/prompt-for-user-input-async.js        
lerna notice 880B  package.json                              
lerna notice 845B  plugin-templates/default/package.json     
lerna notice 1.1kB LICENSE.md                                
lerna notice 2.9kB README.md                                 
lerna notice === Tarball Details === 
lerna notice name:          create-figma-plugin                     
lerna notice version:       0.0.60                                  
lerna notice filename:      create-figma-plugin-0.0.60.tgz          
lerna notice package size:  4.8 kB                                  
lerna notice unpacked size: 13.4 kB                                 
lerna notice shasum:        766551f2d4f66edf1a467355f9c35f88efadeab1
lerna notice integrity:     sha512-7j4n5PwYGB6T6[...]NtyFMqxb8sM9Q==
lerna notice total files:   12                                      
lerna notice 
lerna http fetch PUT 200 https://registry.npmjs.org/create-figma-plugin 5311ms
lerna success published @create-figma-plugin/build 0.0.60
lerna notice 
lerna notice 📦  @create-figma-plugin/build@0.0.60
lerna notice === Tarball Contents === 
lerna notice 751B  lib/build-async.js          
lerna notice 3.6kB lib/build-bundle-async.js   
lerna notice 1.7kB lib/build-manifest-async.js 
lerna notice 648B  lib/cli.js                  
lerna notice 2.3kB lib/create-webpack-config.js
lerna notice 1.0kB lib/watch.js                
lerna notice 1.3kB package.json                
lerna notice 1.1kB LICENSE.md                  
lerna notice === Tarball Details === 
lerna notice name:          @create-figma-plugin/build              
lerna notice version:       0.0.60                                  
lerna notice filename:      create-figma-plugin-build-0.0.60.tgz    
lerna notice package size:  4.1 kB                                  
lerna notice unpacked size: 12.4 kB                                 
lerna notice shasum:        dee83715fd64167d36a0406580561d5cfac612e9
lerna notice integrity:     sha512-ccVbf58oktGP0[...]nN47t4bhsLkhQ==
lerna notice total files:   8                                       
lerna notice 
lerna http fetch PUT 200 https://registry.npmjs.org/@create-figma-plugin%2fbuild 10316ms
Successfully published:
 - @create-figma-plugin/build@0.0.60
 - @create-figma-plugin/common@0.0.60
 - create-figma-plugin@0.0.60
 - @create-figma-plugin/ui@0.0.60
 - @create-figma-plugin/utilities@0.0.60
lerna success published 5 packages
✨  Done in 30.51s.
```

**Comment by yuanqing** [comment]
Looks like an issue with the registry. All modules are at their correct latest versions now.
